compile.sh

```diff
@@ -0,0 +1,3 @@
+gcc -std=c17 -I/home/johannes/src/volk/include -I/home/johannes/src/volk/build/include -L/home/johannes/src/volk/build/lib -x c main.c -o mainvolkgnuc -lm -lvolk
```

Comment:
Guessing you didn't mean to include this file ...

Reply:
ah ... guessing you're using this & `main.c` and `main.cc` as part of the draft for testing purposes, and you'll remove these once the PR is out of draft ... yes?

Reply:
Yes. This is a temporary file I intend to remove as soon as I figured out all the details of how to update the interface.